### DISCUSSION TITLE
这个项目里的绝大多数烂尾楼最后的结果就是烂一生，现在就剩把烂尾楼给炸掉了

### DISCUSSION THREAD

**Discussion by ghost** [WeNeedHome/SummaryOfLoanSuspension#1053] [General]
这个项目里的绝大多数烂尾楼最后的结果就是烂一生，现在就剩把烂尾楼给炸掉了，地方政府已经没有能力来解决烂尾楼这个烂摊子了，以后社会底层和农民有苦日子过了。
![image](https://user-images.githubusercontent.com/110222065/181878410-0f1ab4fb-4a44-4999-8318-3d97cbaa340d.png)

**Comment by Accelerator6666** [comment]
说到底不就是根子上的问题？不想解决，但是又不想让人提，直接归到政治问题上去。
谁敢提出来，就是立场问题，整死你。
有样学样，苏联当初也是这样，切尔诺贝利炸成那样，科学家提出问题，委员会直接先问科学家你的立场是什么，有异议就是共产主义全家桶先来一遍，你说这问题还能解决么？

**Comment by TALK1949** [comment]
当年日本泡沫破裂，其实跟现在的中国一样，留下了一堆烂尾楼，稍微了解一下当时日本的历史就能总结出经验，不过墙内往往关注的点是美国打压别国这种宣传，根本就抓不住重点。

**Reply by oceanzhang860411** [reply]
廣場協議簽訂是一個時間上的標志，并不是政策上的標志，協議是允許美元貶值，日元升值。在開始執行的時候，日本是占盡好處的。但是日本自己把國内的房產價格炒上天了，這你怪誰。當然，共匪比日本更能炒，北上廣的房價收入比已經達到了45-55年的程度，比起當年日本的15-19年，那真是不怕死。
